### ISSUE TITLE
Workspaces / Orgs have automatically deleted

### ISSUE THREAD

**Issue by donenocode** [eclipse-che/che#17599]
## Summary
I have an instance of multiuser Che 6.19.0 running on Openshift 3.11 (running on CentOS).
This morning a number of workspaces and orgs (around 137) have been automatically removed by Che.
The Che pod logs state:
_Workspace 'workspace1234567812345678' removed by user 'undefined'_

And a number of other messages regarding email notifications and such.

Not all workspaces have been removed and my guess is these are older workspaces, but are still required.

The question I have is what triggers this behaviour?
I could restore the system from backup but if this is for instance a time-based trigger then it will just delete them again?
Is there an environment variable that controls this?

## Relevant information
Che 6.19.0
OpenShift 3.11 (on CentOS)
Multiuser

**Comment by skabashnyuk** [comment]
@donenocode could be that these workspaces are temporary workspaces create by the factory? Technically speaking they exist only when they are running.

**Comment by donenocode** [comment]
Thanks for the response.
These workstation are all 'owned' by 1 user, created using the workspace api (SERVER-NAME/api/workspace).

This user is used to create workspaces for students that have limited permissions (for instances they can start/stop/edit but they cannot delete).
The users owns 180+ workspaces, and not all have been removed.
I have restored a backup and this is now working fine, so it would appear the time-limit is not correct.
